amethyst_renderer/src/pass/shaders/fragment/flat.glsl

```diff
@@ -27,5 +27,5 @@ vec2 tex_coords(vec2 coord, vec2 u, vec2 v) {
 }
 
 void main() {
-    color = texture(albedo, tex_coords(vertex.tex_coord, albedo_offset.u_offset, albedo_offset.v_offset));
+    color.rgb = texture(albedo, tex_coords(vertex.tex_coord, albedo_offset.u_offset, albedo_offset.v_offset));
```

Comment:
Why did we remove setting the alpha channel here? That's gonna disable blending on both Flat passes.

Reply:
Yikes, my mistake. Fixing.

Reply:
Apologies, careless mistake.